presto-native-execution/Makefile

```diff
@@ -42,6 +42,12 @@ endif
 ifneq ($(PRESTO_STATS_REPORTER_TYPE),)
   EXTRA_CMAKE_FLAGS += -DPRESTO_STATS_REPORTER_TYPE=$(PRESTO_STATS_REPORTER_TYPE)
 endif
+ifneq ($(PRESTO_MEMORY_CHECKER_TYPE),)
+  EXTRA_CMAKE_FLAGS += -DPRESTO_MEMORY_CHECKER_TYPE=$(PRESTO_MEMORY_CHECKER_TYPE)
+endif
+
+# Set the memory checker type
+PRESTO_MEMORY_CHECKER_TYPE := LINUX_MEMORY_CHECKER
```

Comment:
When @majetideepak asked to add this to the CI he was referring to adding it to the pipeline build here: https://github.com/prestodb/presto/blob/0c19d4f64bc5821fc63d500262f31768cbd4a539/.circleci/continue_config.yml#L123
This will build and test it. What you've done now is hard coding it into the Makefile to always be enabled.